.github/workflows/linux-eic-shell.yml

```diff
@@ -233,6 +233,60 @@ jobs:
           noverlaps="$(grep -c ovlp doc/overlap_check_tgeo.out || true)"
           if [[ "${noverlaps}" -gt "0" ]] ; then echo "${noverlaps} overlaps found!" && false ; fi
 
+  check-geometry-fast:
+    runs-on: ubuntu-latest
+    needs: build-test
+    if: github.ref == 'refs/heads/main'
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/download-artifact@v3
+      with:
+        name: build-fast-eic-shell
+        path: install/
+    - uses: cvmfs-contrib/github-action-cvmfs@v3
+    - uses: eic/run-cvmfs-osg-eic-shell@main
+      with:
+        platform-release: "jug_xl:nightly"
+        setup: install/setup.sh
+        run: |
+          mkdir -p doc
+          checkGeometry -f true -c ${DETECTOR_PATH}/${DETECTOR_CONFIG}.xml | tee geometry_check_fast.out
```

Comment:
If this is "fast", then it should be `-f false` here?

Reply:
Yeah, probably. Originally the goal was to have this work on the fast geometry, no fibers and such, but it doesn't run faster than the full geometry. So yeah, we'd probably switch to false.

Reply:
Or remove, if it's currently identical.

Reply:
Removed.